content/blog/2019/12/2019-12-06-introducing-aws-secrets-manager-credentials-provider-plugin.adoc

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: Introducing the AWS Secrets Manager Credentials Provider for Jenkins
+tags:
+- pipeline
+- plugins
+author: chriskilding
+---
+
+API keys and CI secrets are difficult to handle safely, and probably something you avoid thinking about. In this post I'll show how the new https://github.com/jenkinsci/aws-secrets-manager-credentials-provider-plugin[AWS Secrets Manager Credentials Provider] plugin allows you to marshal your secrets into one place, and use them securely from Jenkins.
```

Comment:
```suggestion
API keys and secrets are difficult to handle safely, and probably something you avoid thinking about. In this post I'll show how the new plugin:aws-secrets-manager-credentials-provider[AWS Secrets Manager Credentials Provider] plugin allows you to marshal your secrets into one place, and use them securely from Jenkins.
```

Reply:
Is `plugin:` a special asciidoc directive that you guys use?

Reply:
yes, https://github.com/jenkins-infra/asciidoctor-jenkins-extensions